### ISSUE TITLE
Add configuration for a gitlab

### ISSUE THREAD

**Issue by mjpitz** [depscloud/cleanroom#2]
Since we have an integration with GitLab, we should test it to ensure compatibility.

https://docs.gitlab.com/charts/

**Comment by MadhuvanthG** [comment]
So this is to only ensure that our system is able to fetch information from Gitlab and process it, is that correct? A compatibility test like you've mentioned.

**Comment by mjpitz** [comment]
Yup. Having worked on the predecessor during the GitLab API v3 to v4 migration, I know we'll have to handle another similar migration at some point.
